org.eclipse.lsp4e/src/org/eclipse/lsp4e/outline/SymbolsLabelProvider.java

```diff
@@ -252,7 +252,7 @@ public String getText(Object element) {
 	@Override
 	public StyledString getStyledText(Object element) {
 
-		if (element == LSSymbolsContentProvider.COMPUTING || element instanceof PendingUpdateAdapter) {
+		if (element instanceof PendingUpdateAdapter) {
```

Comment:
would it be good for UI consistency to replace LSP4E custom message with 
```
if (element instanceof PendingUpdateAdapter pendingUpdate) {
  pendingUpdate.getLaber(null);
}
```

Reply:
I think it might even better for consistency to use directly a `WorkbenchLabelProvider` as super and return `super.getStyledText()` in case the element is not a LS symbol.
But the current patch is clearly a good improvement already. Let's merge.